Client/mods/deathmatch/logic/luadefs/CLuaClientDefs.cpp

```diff
@@ -23,7 +23,9 @@ void CLuaClientDefs::LoadFunctions()
                                                                              {"isChatInputBlocked", ArgumentParser<IsChatInputBlocked>},
                                                                              {"clearDebugBox", ArgumentParser<ClearDebug>},
                                                                              {"isMTAWindowFocused", ArgumentParser<IsMTAWindowFocused>},
-                                                                             {"isCapsLockEnabled", ArgumentParser<IsCapsLockEnabled>}};
+                                                                             {"isCapsLockEnabled", ArgumentParser<IsCapsLockEnabled>},
+                                                                             {"setCursorColor", ArgumentParser<SetCursorColor>},
+                                                                             {"getCursorColor", ArgumentParser<GetCursorColor>}};
```

Comment:
Can you also add `resetCursorColor` to make it normal (255,255,255,1) again?

Reply:
It doesn't make sense. Just like there is no ``resetCursorAlpha`` function, there shouldn't be a ``resetCursorColor`` because it would essentially be just a call to ``setCursorColor(255, 255, 255)``

Reply:
Fair enough. But the wiki should properly document the default color
And perhaps if no arguments are passed to the set function it resets to default?

Reply:
Arguments should be optional
```lua
setCursorColor( [ int r = 255, int g = 255, int b = 255 ] )
``` so, yes
```lua
setCursorColor()
```
should set default white color